tests/unit/test_exclude_capa.py

```diff
@@ -0,0 +1,17 @@
+"""
+Unit test for Exclude CAPA Search Filter Generator
+
+Unit test in order to make sure that the exclude dictionary with
+the ExcludeCapaSearchFilterGenerator contains `content_type` key
+"""
+import unittest
+
+from opencraft_courseware_search_filters import ExcludeCapaSearchFilterGenerator
+
+class TestExcludeCapaSearchFilterGenerator(unittest.TestCase):
+    def test_exclude_dictionary_for_content_type_key(self):
+        search_filter = ExcludeCapaSearchFilterGenerator()
+        self.assertIn("content_type", search_filter.exclude_dictionary())
```

Comment:
Since `ExcludeCapaSearchFilterGenerator`'s job is to exclude CAPA content types, you can even go this far:

```suggestion
        self.assertEqual("CAPA", search_filter.exclude_dictionary()["content_type"])
```

Reply:
Oh! That's actually a much better assertion, I should've paid attention to that. Thanks :smile: :+1: